### PR TITLE
FE_RannacherTurek virtual function overloads

### PR DESCRIPTION
**Pull request by pesser**
Implemented signatures of interpolate functions that were previously hidden as pointed out in #1527

### CODE REVIEW

File: cmake/setup_compiler_flags_gnu.cmake [hunk 1]
```diff
@@ -64,6 +64,7 @@ ENABLE_IF_SUPPORTED(DEAL_II_CXX_FLAGS "-Wwrite-strings")
 ENABLE_IF_SUPPORTED(DEAL_II_CXX_FLAGS "-Wsynth")
 ENABLE_IF_SUPPORTED(DEAL_II_CXX_FLAGS "-Wsign-compare")
 ENABLE_IF_SUPPORTED(DEAL_II_CXX_FLAGS "-Wswitch")
+ENABLE_IF_SUPPORTED(DEAL_II_CXX_FLAGS "-Woverloaded-virtual")
```
Comment (bangerth) [inline cmake/setup_compiler_flags_gnu.cmake]: Can you please break this commit out into its own pull request?

File: source/fe/fe_rannacher_turek.cc [hunk 2]
```diff
@@ -62,7 +63,7 @@ std::string FE_RannacherTurek<dim>::get_name() const
   std::ostringstream namebuf;
   namebuf << "FE_RannacherTurek"
           << "<" << dim << ">"
-          << "(" << this->degree << ", " << this-n_face_support_points << ")";
+          << "(" << this->degree << ", " << this->n_face_support_points << ")";
```
Comment (bangerth) [inline source/fe/fe_rannacher_turek.cc]: Ha, how incredible that that actually compiled previously! :-)

**Comment by bangerth** [comment]
Looks good. Please separate the two commits into their own pull requests and let us know when you're done.

**Comment by pesser** [comment]
Ok, compiler flag is now in #1621

File: source/fe/fe_rannacher_turek.cc (hunk 2 above)
Comment (pesser) [inline source/fe/fe_rannacher_turek.cc]: yes it was a nice name for the element, too ;)
